.ci/test-install-docs.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+# Detect if any installation guides changed. If so, run the
+# kata manager to "execute" the install guide to ensure the
+# commands it specified result in a working system.
+check_install_guides()
+{
+	local -r install_guide_suffix="-installation-guide.md"
+
+	# List of filters used to restrict the types of file changes.
+	# See git-diff-tree(1) for further info.
+	local filters=""
+
+	# Added file
+	filters+="A"
+
+	# Copied file
+	filters+="C"
+
+	# Modified file
+	filters+="M"
+
+	# Renamed file
+	filters+="R"
+
+	# Unmerged (U) and Unknown (X) files. These particular filters
+	# shouldn't be necessary but just in case...
+	filters+="UX"
+
+	# List of changed files
+	local files=$(git diff-tree \
+		--name-only \
+		--no-commit-id \
+		--diff-filter="${filters}" \
+		-r \
+		origin/master HEAD || true)
+
+	# No files were changed
+	[ -z "$files" ] && return
```

Comment:
As this is a CI - maybe a message here for the logs noting we finished as there was nothing to do (rather than a silent quit, which I predict some day somebody will have to go 'debug' :-) )

Reply:
Fixed.